app/views/donuts/index.html.erb

```diff
@@ -8,4 +12,4 @@
   </ul>
 <% end %>
 
-<%= link_to "Add New Donut", new_donut_path %>
+<%= link_to "Add New Donut", new_donut_path, class: 'add-link' %>
```

Comment:
I might suggest changing this to 'add-donut-link' so that it cannot be confused with a button that creates a link